modules/i18n/src/translations/en-US/portals/console.ts

```diff
@@ -9461,12 +9461,12 @@ export const console: ConsoleNS = {
             parentOrgInvitations: {
                 addUserWizard: {
                     heading: "Invite Parent User",
-                    description: "Invite a user from the parent organization.",
+                    description: "Invite users from the parent organization.",
                     hint: "Invited users are managed by the parent organization.",
                     username: {
                         label: "Username",
                         placeholder: "Enter the username",
-                        hint: "Username should belong to a user from the parent organization.",
+                        hint: "Add the username of a parent user and press enter. Repeat to include multiple users.",
```

Comment:
Check with the Doc team if we can improve this content

Reply:
Approved by doc team